### DISCUSSION TITLE
Request support for self-hosted umami vs. Cloud umami

### DISCUSSION THREAD

**Discussion by QYG2297248353** [outline/outline#7280] [Ideas]
https://umami.is/
![image](https://github.com/user-attachments/assets/10986fde-2fed-477f-9f9e-50aedc99f51a)

https://umami.is/docs/collect-data
![image](https://github.com/user-attachments/assets/c1467867-9531-4102-8829-d27d3fa65933)

It can be configured by host domain name + script name + site ID

**Comment by tommoor** [comment]
If someone wants to add support based on the existing Matamo and Google integrations, go for it!
